src/IW/Search.hs

```diff
@@ -18,10 +18,17 @@ fetchHaskellReposHW = searchRepos "language:haskell help-wanted-issues:>0"
 fetchHaskellReposGFI :: IO (Either Error (SearchResult Repo))
 fetchHaskellReposGFI = searchRepos "language:haskell good-first-issues:>0"
 
--- | Fetch all issues with Haskell language and label "help-wanted"
+-- | Fetch all open issues with Haskell language and the labels passed in to the function
+fetchOpenIssuesWithLabels :: [Text] -> IO (Either Error (SearchResult Issue))
+fetchOpenIssuesWithLabels labels = searchIssues $ "language:haskell is:open " <> labelQualifiers
+  where
+    labelQualifiers :: Text
+    labelQualifiers = mconcatMap (\x -> " label:\"" <> x <> "\" ") labels   
```

Comment:
Hmm, interesting :thinking: I had an impression that `concatMap` should work here...
However, you don't need to depend on the `extra` package. You can just use `foldMap` instead of `mconcatMap`.

Reply:
Okay I can change. concatMap didn’t work because it’s Text. If it was String I think it would work though. I’ll checkout the foldMap function. Thank you!